core-api/src/main/java/com/optimizely/ab/notification/ActivateNotificationListener.java

```diff
@@ -51,6 +58,17 @@ public final void notify(Object... args) {
         onActivate(experiment, userId, attributes, variation, logEvent);
     }
 
+    @Override
+    public final void notify(ActivateNotification message) {
```

Comment:
If it doesn’t break public API compatibility, it would nice to not be overloading `Object.notify` anymore...

Reply:
Ah, good point I can change that. Suggestion? `handle`? since the new interface is `NotificationHandler<T>`